controlplane/kubeadm/main.go

```diff
@@ -51,8 +54,11 @@ func init() {
 
 	_ = clientgoscheme.AddToScheme(scheme)
 	_ = clusterv1alpha3.AddToScheme(scheme)
+	_ = clusterv1alpha4.AddToScheme(scheme)
```

Comment:
Same comment as in another main.go - I think we only need the alpha4 APIs added to the scheme